app/views/episodes/_form.html.erb

```diff
@@ -1,22 +1,23 @@
-<%= form_with(model: episode, html: {autocomplete: "off"}, data: {controller: "unsaved", unsaved_changed_value: episode.changed?, unsaved_confirm_value: t(".confirm")}) do |form| %>
+<% model = episode.new_record? ? [episode.podcast, episode] : episode %>
```

Comment:
What does the `[episode.podcast, episode]` array accomplish in the form?

Also curious why it's defined here instead of inside a helper or a controller.

Reply:
This is varying the form url/method, depending if this is a new episode (`POST /podcasts/123/episodes`) or an existing one (`PATCH /episodes/abcd1234`).

I added it to a helper, but that seemed like unnecessary obfuscation.  And the `model:` option was the easiest way to make it work in both create/update.